### ISSUE TITLE
textDocument/definition finds the last variable declaration instead of the earliest one

### ISSUE THREAD

**Issue by rcjsuen** [rcjsuen/dockerfile-language-server-nodejs#141]
```Dockerfile
FROM node
ARG var=value
ARG var=value2
RUN echo $var
RUN echo $var
```
If you select the `$var` on the two `RUN` lines and invoke the `textDocument/definition` request, it will jump to the second `ARG` line. It should jump to the first `ARG` line instead.

**Comment by rcjsuen** [comment]
Actually, looks like it finds the last one instead. The `textDocument/definition` request returns the second `ARG` instruction at the bottom instead of the first.
```Dockerfile
FROM node
ARG var
RUN echo $var
ARG var
```

**Comment by rcjsuen** [comment]
Fixed by 98601e92433df1a5ec808dc2df98f5927218892a. The commit message is wrong and it fixes this issue, not #140.
